pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/services/ReportPathGeneratorService.kt

```diff
@@ -46,10 +47,16 @@ class ReportPathGeneratorService(private val project: Project) {
         return Path.of("$projectBasePath/.history")
     }
 
+    /**
+     * Checks if either the given build type or the debug build type exists.
+     * @return the path to the report directory with the build type or without
+     */
     private fun checkForDebugBuiltType(): Path {
-        var path = getReportPath()
-        if (Files.exists(Path.of("$path/debug"))) {
-            path = Path.of("$path/debug")
+        val path = getReportPath()
+        if (buildType != null && Files.exists(Path.of("$path/$buildType"))) {
+            return Path.of("$path/$buildType")
+        } else if (Files.exists(Path.of("$path/debug"))) {
```

Comment:
I think you for got to test for release buidtype here right?

Reply:
I just kept it as was before. If a build type is set - try that one. Otherwise try debug (what we were doing before by default) and then use no build type

Reply:
Ok than I would change the message to 

```
       val multilineText = "<html>The Android build type of which you want to use for the pitest results from.<br/>If kept empty, <c>debug</c> is tried if it is an Android project.</html>"
```